pages/components/Toast/toast.css

```diff
@@ -0,0 +1,46 @@
+
+.toast-display-wrapper{
+    padding: 2rem;
+    border: 1px solid var(--white);
+}
+
+.toast-display-wrapper>*+*{
+    margin-left: 2rem;
+}
+
+.toast{
+    display: flex;
+    align-items: center;
+    flex-basis: 26rem;
+    border-radius: 0.5rem;
+    color: var(--pink8);
+    font-size: 1.3rem;
+    padding: 0.4rem;
```

Comment:
for smaller value use px if the value is less than or equal to 10 px

Reply:
okeiy ,thanks ! will do it from next time.